3dgeotop/gt2pc.xsh

```diff
@@ -63,25 +63,26 @@ if args.clipsrclayer: ogr_args.extend(["-clipsrclayer", args.clipsrclayer])
 
 ogr_args.extend(['-a_srs',f"EPSG:{ args.srs_in }", '-oo', 'AUTODETECT_TYPE=YES', '-oo', 'X_POSSIBLE_NAMES=x', '-oo', 'Y_POSSIBLE_NAMES=y', '-oo', 'Z_POSSIBLE_NAMES=z'])
 ogr_args.extend(['-dialect', 'sqlite'])
-ogr_args.extend(['-sql', f"select x,y,z * { args.multiplier }, lut.klasse / 10, lut.R, lut.G, lut.B FROM { os.path.splitext(os.path.basename(args.file))[0] } as sample { args.join_type } JOIN '{ args.color_table }'.lut as lut ON sample.lithoklasse = lut.klasse { where }" ])
+ogr_args.extend(['-sql', f"select x,y,z * { args.multiplier }, 1, lut.R, lut.G, lut.B, lut.klasse FROM { os.path.splitext(os.path.basename(args.file))[0] } as sample { args.join_type } JOIN '{ args.color_table }'.lut as lut ON sample.lithoklasse = lut.klasse { where }" ])
```

Comment:
@MarcoDuiker Je had hier bij `lut.klasse` eerder `+ 20` staan. Was dat de bedoeling?

Reply:
Jazeker, dat was en is de bedoeling.

1 t/m 19 hebben volgens de las-standaard een speciale betekenis en zijn daarmee gereserveerd. Door er 20 bij op te tellen blijven we weg uit de gereserveerde range.

Reply:
Oke! Ik heb het weer hersteld.
Heb je een idee hoe we dit voor de freshem klassen kunnen doen? Daar is één klasse met waarde `0`.

Reply:
Volgens de [LAS-standaard](https://www.asprs.org/wp-content/uploads/2019/03/LAS_1_4_r14.pdf) is 0 t/m 31 gereserveerd (zie screenshot). 
Zullen we anders +100 doen? Dan blijven de classificatiewaarden nog wel enigszins intact
![image](https://github.com/ProvincieZeeland/3DToolbox/assets/122441727/759d3404-cd53-41cd-b771-70e612e274f5)

Wat betreft FRESHEM: de Classification-waarde in LAS is 1 byte, dus waardes van >255 worden niet opgeslagen. De huidige LUT van FRESHEM loopt van 0 tot 15000. Zullen we die klasses (die denk ik uit chloridewaardes komen?) mappen naar oplopende klassenummers? Zoiets dus: 

```
chloride,R,G,B,klasse
0,0,0,127,0
150,0,0,250,1
300,0,88,255,2
500,0,196,255,3
750,30,226,221,4
1000,60,255,186,5
...
```

Reply:
+100 vind ik een prima voorstel

Reply:
Geïmplementeerd in https://github.com/ProvincieZeeland/3DToolbox/pull/4/commits/ee719b03d2843ac09aa65b39bc6cdb6d62ea5929